rts/System/SpringMath.cpp

```diff
@@ -436,3 +436,8 @@ float3 hs2rgb(float h, float s)
 
 	return col;
 }
+
+float distanceToPlane(const float3 point, const float4& plane)
```

Comment:
float3::dot() does the same.

Reply:
Also `const float3& point` not `const float3 point`

Reply:
True, I'll use that